CMakeLists.txt

```diff
@@ -180,6 +177,13 @@ elseif(TOFINO_TARGET)
     include(tofino-driver)
 endif()
 
+if(WITH_OVSP4RT)
+    if(OVS_INSTALL_DIR STREQUAL "")
+        message(FATAL_ERROR "OVS_INSTALL_DIR not defined!")
```

Comment:
Be consistent in messaging - message(FATAL_ERROR "OVS_INSTALL_DIR (OVS_INSTALL) not defined!")? Similar to line 40 and 42.